CorpusSearch/ClientApp/src/components/Home.tsx

```diff
@@ -159,7 +159,7 @@ export const Home = () => {
                         </span>
                     </>
                 }
-                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects</span>
+                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects. Send submissions to <a href="mailto:corpus-submissions@gaelg.im">corpus-submissions@gaelg.im</a></span>
```

Comment:
Thoughts? We definitely want the line-break. Implementer's choice for the wording

```suggestion
                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects.</span>
                <br/>
                <span>If we're missing anything, please let us know at <a href="mailto:corpus-submissions@gaelg.im">corpus-submissions@gaelg.im</a>.</span>
```

<img width="1296" alt="Screenshot 2024-03-30 at 20 15 31" src="https://github.com/david-allison/manx-corpus-search/assets/62114487/dba093f3-b00f-48a8-94e0-c88851c6f0ce">

----

My main goal here is to ensure that we don't have any 'unknown unknown' documents. The secondary goal is to reduce the effort to the minimum possible, we're more likely to have people email us if they feel they won't need to reformat/transcribe/translate a document, and we can add this to a work queue

Possibly: `submissions` -> `tips`/`leads`

Reply:
Makes sense. Your change is definitely an improvement. "Submission" sounded more like a contest to me. I actually had a line break to start but took it out for some reason.

Reply:
When you're free, feel free to push/force push to your `mailto` branch and it'll update here

[force push to PR branches strongly encouraged, but I can squash/rebase on my end]

Reply:
Heh, I didn't even notice your comment until after I squashed and force pushed.